packages/arb-bridge-eth/contracts/challenge/Challenge.sol

```diff
@@ -222,15 +198,15 @@ contract Challenge is Cloneable, IChallenge {
         );
     }
 
-    function constraintWinExecution(
+    function proveContinuedExecution(
         bytes32[] calldata _merkleNodes,
         uint256 _merkleRoute,
         uint256 _challengedSegmentStart,
         uint256 _challengedSegmentLength,
         bytes32 _oldEndHash,
         uint256 _gasUsedBefore,
         bytes32 _assertionRest
-    ) external executionChallenge onlyOnTurn {
+    ) external onlyOnTurn {
         require(_challengedSegmentLength > 1, "TOO SHORT");
```

Comment:
There was some conversation with the auditors about potentially needing to drop this requirement. In the new formulation, I'm not totally sure if that's the case, but I think it might still be true